Felienne/hedy
# [FIX] Removes print margin line for ace editor

Description**
This PR removes the print line from the ace editor.

**Fixes**
This PR fixes #3340.

**How to test**
Make a long program and verify that the vertical line after 80 characters is no longer shown.
